packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -70,30 +66,24 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   link: {
     textDecorationLine: 'underline'
   },
+  shadow: {
+    shadowColor: 'black',
+    shadowOpacity: 0.2,
+    shadowRadius: 5,
+    shadowOffset: { width: 0, height: 3 }
+  },
   tail: {
     display: 'flex',
     position: 'absolute',
-    bottom: 47
+    bottom: -0.4
   },
   tailIsAuthor: {
-    right: -spacing(3)
+    right: -18
```

Comment:
It might be nice to just pull these values into constants that define what they are (`const tailSize = 18` and then `right: tailSize`).
It makes it more obvious _why_ we are offsetting and what element or measurement the offset is based on.